wallet-api/src/Ledger/Index.hs

```diff
@@ -202,7 +207,8 @@ validationData h tx = rump <$> ins where
         , pendingTxForge = txForge tx
         , pendingTxFee = txFee tx
         , pendingTxSlot = h
-        , pendingTxOwnHash    = ()
+        , pendingTxIn = head inputs
+        -- this is changed accordingly in `checkMatch` during validation
```

Comment:
This is a bit inelegant, but no different to how it was before.

Reply:
Yeah, I don't like it very much as well. I tried to change it, and didn't like the result either: more code changes, duplicated PendingTx initialization. Anyway, we guarantied to have at least on input.

If you suggest I will change it, but to me it appears "good enough"™️

Reply:
Yeah, it's fine for now.